[dx-732] release note 5.2.1, no docker link (#3306)
TykTechnologies/tyk-docs: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.2.md

---
title: Tyk Dashboard 5.2 Release Notes
date: 2023-09-27T15:49:11Z
description: "Release notes for Tyk Dashboard version 5.2"

> tags: ["Release notes", "v5.2", "5.2.0", "5.2", "changelog"]

Reviewer: Do you plan to add all versions in here?

Author: was thinking about that, is this a problem?

Reviewer: I don't have an opinion.... just that I spotted it has different variations of 5.2, but not 5.2.1...

---

**Licensed**

> ##### This page contains all release notes for version 5.2.X displayed in a reverse chronological order

Reviewer: ```suggestion
##### This page contains all release notes for version 5.2.X displayed in reverse chronological order
```

Author: no idea why  this one is outdated @andyo-tyk

> ##### This page contains all release notes for version 5.2.X displayed in a reverse chronological order

Author: ```suggestion
##### This page contains all release notes for version 5.2.X displayed in reverse chronological order
```

### Support Lifetime

> Our minor releases are supported until our next minor comes out. This would be R5.3 scheduled in Q4 if this goes ahead as planned. If not, 5.2 will remain in support until our next LTS version comes out in March 2024.

Author: This was released as part of 5.2.0 per @allaneyles request and I just moved it to the top. FYI @dcs3spp

> Our minor releases are supported until our next minor comes out. This would be R5.3 scheduled in Q4 if this goes ahead as planned. If not, 5.2 will remain in support until our next LTS version comes out in March 2024.

Reviewer 2: Updated support lifetime statement for Tyk dashboard

### Upgrading Tyk
Please refer to the [upgrading Tyk]({{< ref "upgrading-tyk" >}}) page for further guidance with respect to the upgrade strategy.

---

## 5.2.1 Release Notes 

### Breaking Changes
This release has no breaking changes.

### Deprecation
There are no deprecations in this release.

### Upgrade instructions

> If you are on a 5.2.0 please upgrade asap and if you are on an older version skip 5.2.0 and upgrade directly to this release. 

Reviewer: ASAP should be capitalised

### Release Highlights
This release primarily focuses on bug fixes. 
For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-v5.2.0">}}) below.

### Downloads
[docker image to pull](https://hub.docker.com/layers/tykio/tyk-dashboard/v5.2.0/images/bla)

### Changelog {#Changelog-v5.2.1}

#### Added (1 change)
- Added support to Tyk Dashboard API so that Tyk Sync can fully support Tyk OAS API Definitions; this will be enabled from Tyk Sync version 1.4.1.

#### Fixed (3 changes):
- Fixed a bug in *Tyk Dashboard* UI where pagination in the APIs screen was breaking for API of type GraphQL/UDG and resulted in the page failing to load data and displaying a 'No data to display' message.

> - Fixed an issue where the "Add GraphQL Operation" checkbox in the GraphQL data source configuration screen couldn't be disabled, even when no operation was added. Now, its state can be adjusted based on the presence of GraphQL operations and variables.

Author: what's the difference

> - Fixed an issue where the "Add GraphQL Operation" checkbox in the GraphQL data source configuration screen couldn't be disabled, even when no operation was added. Now, its state can be adjusted based on the presence of GraphQL operations and variables.

Reviewer 2: Suggestion uses single quotation style to be consistent with other changelog items

---
title: Tyk Dashboard 5.2 Release Notes
date: 2023-09-27T15:49:11Z
description: "Release notes documenting updates, enhancements, and changes for Tyk Dashboard versions within the 5.2.X series."
tags: ["Tyk Dashboard", "Release notes", "v5.2", "5.2.0", "5.2", "changelog", "5.2.1"]
---

**Licensed Protected Product**

**This page contains all release notes for version 5.2.X displayed in reverse chronological order**

### Support Lifetime
Minor releases are supported until our next minor comes out. There is no 5.3 scheduled in Q4. Subsequently, 5.2 will remain in support until our next LTS version comes out in March 2024.

---

## 5.2.1 Release Notes 

> ##### Release Date 10 Oct 2023 <<update>>

Reviewer 2: ```suggestion
##### Release Date 10 Oct 2023
```

#### Breaking Changes
This release has no breaking changes.

#### Deprecation
There are no deprecations in this release.

#### Upgrade instructions
If you are on a 5.2.0 we advise you to upgrade asap and if you are on an older version skip 5.2.0 and upgrade directly to this release. 

#### Release Highlights
This release primarily focuses on bug fixes. 
For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-v5.2.0">}}) below.

#### Downloads
<<[docker image to pull](https://hub.docker.com/layers/tykio/tyk-dashboard/v5.2.0/images/bla)>>

#### Changelog {#Changelog-v5.2.1}

##### Added (1 change)
- Added support to Tyk Dashboard API so that Tyk Sync can fully support Tyk OAS API Definitions; this will be enabled from Tyk Sync version 1.4.1.

> ##### Fixed (2 changes):

Reviewer 3: ```suggestion
##### Fixed (3 changes):
```

> <<[docker image to pull](https://hub.docker.com/layers/tykio/tyk-dashboard/v5.2.0/images/bla)>>

Reviewer 2: ```suggestion
- [docker image to pull](https://hub.docker.com/layers/tykio/tyk-dashboard/v5.2.1/images/sha256-2f9d8af0e57f7fe4afb618dcf34772c001104dc0ec62a27541d12dc9ae90d5c8?context=explore)
```

- Fixed a bug in the *Tyk Dashboard* UI where pagination in the APIs screen was breaking for API of type GraphQL/UDG. This resulted in the page failing to load data and displaying a 'No data to display' message.

- Fixed an issue where the 'Add GraphQL Operation' checkbox in the GraphQL data source configuration screen couldn't be disabled, even when no operation was added. Now, its state can be adjusted based on the presence of GraphQL operations and variables.

---

## 5.2.0 Release Notes 

##### Release Date 29 Sep 2023

#### Breaking Changes

This release has no breaking changes.

#### Release Highlights

We're thrilled to bring you some exciting enhancements and crucial fixes to improve your experience with Tyk Dashboard. For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-v5.2.0">}}) below.

> ##### Configure Caching For Each API Endpoint

Reviewer 4: @letzya @andyo-tyk we added just the timeout field, this title can be a bit missleading?

Reviewer 4: First one looks good to me (if it's not too long).

- Fixed a potential security vulnerability where *static* or *dynamic mTLS* requests with expired certificates could be proxied upstream.

- Fixed an issue in which duplicate API names and listen paths could be created. Configurations are now unique.

- Fixed an issue in the *API Activity* dashboard where users were unable to view request analytics for a specific date. Subsequently, users can now make informed decisions based on access to this data. 

- Fixed an issue where the [Enforced Timeout]({{< ref "planning-for-production/ensure-high-availability/enforced-timeouts" >}}) configuration parameter of an API endpoint accepted negative values, without displaying validation errors. With this fix, users receive clear feedback and prevent unintended configurations.

> - Fixed an issue in *Tyk Dashboard* where duplicate APIs could be created with the same names and listen paths if you clicked multiple times on the *save* button in the API Designer. Now, when you save your new API, you are taken to the list of APIs.

Reviewer 5: ```suggestion
- Fixed an issue in *Tyk Dashboard* where duplicate APIs could be created with the same names and listen paths if you clicked multiple times on the *save* button in the API Designer. Now, this is not possible anymore and there is no risk of creating multiple APIs with the same name.
```

Reviewer 5: Few lines above we're writing that we changed the "go back to APIs list after Save" behaviour, so this last sentence doesn't make sense.
